hypha/apply/users/templates/two_factor/core/backup_tokens.html

```diff
@@ -1,26 +1,33 @@
 {% extends "two_factor/_base_focus.html" %}
-{% load i18n users_tags %}
+{% load static i18n users_tags %}
 
 {% block content %}
-  <p><a href="{% url 'users:account'%}"
-       class="btn btn-link">{% trans "Back to account" %}</a></p>
+  <script src="{% static 'js/clipboard.js' %}"></script>
+  <p class="actions actions-header"><a href="{% url 'users:account' %}" class="link link--bold">{% trans "Back to Account" %}</a></p>
   <h1>{% block title %}{% trans "Backup Codes" %}{% endblock %}</h1>
-  <p>{% blocktrans trimmed %}These codes should be kept in a safe,
-      private place for when you need them. When they are used up,
-      you can generate a new set of backup codes.{% endblocktrans %}</p>
+  <p>{% blocktrans trimmed %}Each of the code can be used only once. When they are used up, you can generate a new set of backup codes.{% endblocktrans %}</p>
 
   {% if device.token_set.count %}
-    <ul>
-      {% for token in device.token_set.all %}
-        <li>{{ token.token }}</li>
-      {% endfor %}
-    </ul>
-    <p>{% blocktrans %}You should now download, or print these codes,
-        and keep them somewhere safe.{% endblocktrans %}</p>
-    <form method="post">{% csrf_token %}{{ form }}
+<textarea readonly cols="8" rows="{{ device.token_set.count }}" id="list-backup-tokens">{% for token in device.token_set.all %}{{ token.token }}{% if not forloop.last %}&#013;&#010;{% endif %}{% endfor %}</textarea>
+    <p class="hide-print">{% blocktrans %}You should now print these codes or copy them to your
+       clipboard and store them in your password manager.{% endblocktrans %}</p>
+
+    <form method="post" class="actions actions-footer">{% csrf_token %}{{ form }}
+    <p>
+        <a href="#" class="btn btn-primary btn-print">{% trans "Print" %}</button>
+        <a href="#"
+            class="btn btn-primary btn-copy-to-clipboard"
+            data-clipboard-target="#list-backup-tokens"
+            data-clipboard-action="copy"
+        >{% trans "Copy to Clipboard" %}</a>
         <button class="btn btn-primary" type="submit">{% trans "Regenerate Codes" %}</button>
-        <a class="btn btn-link link--left-space" href="data:text/plain;charset=UTF-8,{% tokens_text device.token_set.all %}" download="backup_codes.txt">
-        {% trans "Save Codes" %}</a>
+    </p>
+
+    <p class="hide-print">{% blocktrans %}Once done, then click next.{% endblocktrans %}</p>
+    <p>
+    <a class="btn btn-link" href="#">{% trans "Next" %}</a>
```

Comment:
This link goes nowhere yet.

Reply:
Fixed. It will take you back to the user settings.

![image](https://user-images.githubusercontent.com/236356/201075345-b3810149-53e1-401f-8c8d-6b01e8e7a07b.png)